src/Organizations/OrganizationDetails/OrganizationDetailsContainer.js

```diff
@@ -39,8 +41,9 @@ export const OrganizationDetailsContainer = ({
   const [isLoading, setIsLoading] = useState(true);
   const [organizationCategories, setOrganizationCategories] = useState([]);
   const [translatedCategories] = useTranslatedCategories(organizationCategories);
+  const organizationTypesAll = useTypes();
```

Comment:
Please follow to common code style:
`const { orgTypes, isLoading } = useTypes()`

Reply:
adapt to common code style